RS-Project12/CircularQueueUsingArray.java

```diff
@@ -0,0 +1,61 @@
+package stackAndQueueImplementation;
+
+public class CircularQueueUsingArray implements QueueInterface<Integer>{
+    private Integer rear;
+    private Integer front;
+    private Integer[] queue;
+    CircularQueueUsingArray() {
+        queue=new Integer[5];

```

Comment:
one space before and after equals to operator - this line should be written like - queue = new Integer[5];